src/pages/example.tsx

```diff
@@ -11,6 +21,79 @@ export default function Example() {
           Back to homepage
         </Button>
       </Link>
+
+      <Dropdown
+        options={[
+          {
+            value: 1,
+            label: 'Option one',
+          },
+          {
+            value: 2,
+            label: 'Option two',
+          },
+          {
+            value: 3,
+            label: 'Option three',
+          },
+        ]}
+      >
+        <Button icon={FaChevronDown} variant="secondary" iconPosition="right">
+          Menu Dropdown
```

Comment:
The font settings for the menu dropdown doesn't match the other menus. Font seems to small and to bold. Please check this again.

Reply:
https://github.com/MakairaIO/app-boilerplate-nextjs/pull/2#discussion_r1129196048 
Maybe comes from same issue ? right?